config/routes.rb

```diff
@@ -21,12 +21,14 @@
   get "/cookies", to: "static_pages#cookies", as: :cookies
   get "/accessibility_statement", to: "static_pages#accessibility_statement", as: :accessibility_statement
 
-  constraints slug: %r{#{PageSequence::SLUGS.join("|")}} do
-    resources :claims, only: [:new, :create, :show, :update], param: :slug, path: "/claim"
-  end
+  scope path: ":policy", defaults: {policy: "student-loans"}, constraints: {policy: %r{student-loans}} do
+    constraints slug: %r{#{PageSequence::SLUGS.join("|")}} do
+      resources :claims, only: [:new, :create, :show, :update], param: :slug, path: "/"
```

Comment:
Is it worth sending the policy to the controller? (even though we might not use it yet), so we can set the eligibility model at a later date?

Reply:
Yeah the policy is available in the controller via `params[:policy]` at the moment. Were you thinking of something else?

Reply:
Oh yeah, sorry, I missed that 🤦‍♂